src/legacy/ui/public/agg_types/controls/components/from_to_list.tsx

```diff
@@ -0,0 +1,141 @@
+  labelledbyId: string;
+  list: FromToObject[];
+  showValidation: boolean;
+  onBlur(): void;
+  onChange(list: FromToObject[]): void;
+  setValidity(isValid: boolean): void;
+}
+
+function FromToList({ labelledbyId, showValidation, onBlur, ...rest }: FromToListProps) {
+  const fromToListConfig: InputListConfig = {
+    defaultValue: {
+      from: { value: '0.0.0.0', model: '0.0.0.0', isInvalid: false },
+      to: { value: '255.255.255.255', model: '255.255.255.255', isInvalid: false },
+    },
+    defaultEmptyValue: {
+      from: { value: EMPTY_STRING, model: EMPTY_STRING, isInvalid: false },
+      to: { value: EMPTY_STRING, model: EMPTY_STRING, isInvalid: false },
+    },
+    validateClass: Ipv4Address,
+    getModelValue: (item: FromToObject) => ({
+      from: {
+        value: item.from || EMPTY_STRING,
+        model: item.from || EMPTY_STRING,
+        isInvalid: false,
+      },
+      to: { value: item.to || EMPTY_STRING, model: item.to || EMPTY_STRING, isInvalid: false },
+    }),
+    getModel: (models: FromToModel[], index, modelName: 'from' | 'to') => models[index][modelName],
+    getRemoveBtnAriaLabel: (item: FromToModel) =>
+      i18n.translate('common.ui.aggTypes.ipRanges.removeRangeAriaLabel', {
+        defaultMessage: 'Remove the range of {from} to {to}',
+        values: { from: item.from.value, to: item.to.value },
+      }),
+    onChangeFn: ({ from, to }: FromToModel) => {
+      const result: FromToObject = {};
+      if (from.model) {
+        result.from = from.model;
+      }
+      if (to.model) {
+        result.to = to.model;
+      }
+      return result;
+    },
+    hasInvalidValuesFn: ({ from, to }: FromToModel) => from.isInvalid || to.isInvalid,
+    renderInputRow: (item: FromToModel, index, onChangeValue) => (
+      <>
+        <EuiFlexItem>
+          <EuiFieldText
+            aria-labelledby={`visEditorIpRangeFromLabel${labelledbyId}`}
+            isInvalid={showValidation ? item.from.isInvalid : false}
+            placeholder="*"
+            onChange={ev => {
+              onChangeValue(index, ev.target.value, 'from');
+            }}
+            value={item.from.value}
+            onBlur={onBlur}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFieldText
+            aria-labelledby={`visEditorIpRangeToLabel${labelledbyId}`}
+            isInvalid={showValidation ? item.to.isInvalid : false}
+            placeholder="*"
+            onChange={ev => {
+              onChangeValue(index, ev.target.value, 'to');
+            }}
+            value={item.to.value}
+            onBlur={onBlur}
+          />
+        </EuiFlexItem>
+      </>
+    ),
+    validateModel: (validateFn, object: FromToObject, model: FromToModel) => {
+      validateFn(object.from, model.from);
+      validateFn(object.to, model.to);
+    },
+  };
+  const header = (
+    <>
+      <EuiFlexItem>
+        <EuiFormLabel htmlFor={`visEditorIpRangeFromLabel${labelledbyId}`}>
```

Comment:
This prop needs to be the `id` because `aria-labelledby` maps to an id not `for`.
```suggestion
        <EuiFormLabel id={`visEditorIpRangeFromLabel${labelledbyId}`}>
```